src/main/java/org/broadinstitute/consent/http/service/DataAccessParser.java

```diff
@@ -138,7 +139,9 @@ PDAcroForm fillDARForm(Document dar, Map<String, String> researcherProperties, U
                     field.setValue(getDefaultValue(dar.getString(DarConstants.RUS)));
                     break;
                 }
-                case DarConstants.NON_TECH_RUS: {
+                case DarConstants.NON_TECH_RUS:
+                    // Handle legacy "non_tech_rush"
+                case "non_tech_rush": {
```

Comment:
This isn't a typo? The legacy variable name was ```non_tech_rush``` and not ```non_tech_rus```?

Reply:
This was a typo too, good catch.